plutus-playground-client/src/Action.purs

```diff
@@ -71,24 +71,32 @@ simulationNavItem :: forall p. Int -> Int -> Simulation -> Array (HTML p Query)
 simulationNavItem activeIndex index simulation =
   [ button
       [ id_ $ "simulation-nav-item-" <> show index
-      , buttonClasses
+      , classes $ buttonClasses <> [ simulationNavItemNameClass ]
       , onClick $ input_ $ SetSimulationSlot index
       ]
       [ text $ "Simulation #" <> show (index + 1) ]
   , button
       [ id_ $ "simulation-nav-item-" <> show index <> "-remove"
-      , buttonClasses
+      , classes $ buttonClasses <> [ simulationNavItemCloseClass ]
       , onClick $ input_ $ RemoveSimulationSlot index
       ]
       [ icon Close ]
   ]
   where
     buttonClasses =
-      classes ([ btn, simulationNavItemClass ] <> if activeIndex == index then [ btnPrimary ] else [ btnInfo ])
+        [ btn, simulationNavItemClass ]
+        <>
+        if activeIndex == index then [ btnPrimary ] else [ btnInfo ]
```

Comment:
am I missing something here, why isn't `simulationNavItemCloseClass` in `buttonClasses`?

Reply:
Because there's a different class for the "name" button and the "close" button.